spec/lexers/squirrel_spec.rb

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Please add '# frozen_string_literal: true'

Reply:
Just added it. Sorry for the delay.